qbraid/runtime/iqm/_compat.py

```
@@ -0,0 +1,107 @@
+
+
+@lru_cache(maxsize=1)
+def load_iqm_qiskit_symbols() -> SimpleNamespace:
+    """Load qiskit adapter helpers shipped with ``iqm-client[qiskit]``."""
+    try:
+        qiskit_to_iqm = import_module("iqm.qiskit_iqm.qiskit_to_iqm")
+        iqm_qiskit_job = import_module("iqm.qiskit_iqm.iqm_job")
+    except ImportError as err:
+        raise ImportError(
+            "IQM qiskit support requires the optional 'iqm-client[qiskit]' dependencies to be installed."
+        ) from err
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)